### DISCUSSION TITLE
2021-09-01 Weekly Meeting 19 Agenda

### DISCUSSION THREAD

**Discussion by humbletim** [omigroup/omigroup#82] [Weekly Meetings]
Weekly Meeting 19 is happening on Wednesday September 1, 2021 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
   - Welcome new members and attendance
   - [OMI Survey](https://forms.gle/Dxvap4auBGim6JsL9) follow-up
   - glTF Sub Group follow-up
   - 15 min [Retrospective](https://app.mural.co/t/altontech5044/m/altontech5044/1630338172367/94b4adf757f3a4842bf002a6c400c2b94ec99527) jam session
   - Demo Night coordination follow-up
     - https://forms.gle/CknpQQKNi7ywQu9u5

Add ons: 
   - Website call out for section maintainers and community content
   - Discord Roles/Bot follow-ups
   - Deprioritizing the OMI space to meet work and play #44 
   - Working agreements / [playbook lite template](https://app.mural.co/t/altontech5044/m/altontech5044/1630340368839/d4029b3d2dd4efae2dc124995842fe70253a2bfa)

**Comment by mrmetaverse** [comment]
Added links to the retro and new working agreement template. 

I suggest deprioritizing the working agreement follow up section once we have created one working agreement for one working group, it inky works if it's done collaboratively. I also recommend scanning past agendas for items we may have removed since we are copy pasting the prev weeks agendas as a baseline for each new weeks agenda. 

I think the best way to actually talk about what is active, done, or worth prioritizing is to use the backlog in a screen share. If there is time, I propose we take 5 minutes today to groom.

**Reply by mrmetaverse** [reply]
The agenda should almost always coincide with the "active", or "review" tickets.

**Comment by mrmetaverse** [comment]
Started meeting notes here: https://hackmd.io/@mrmetaverse/omi-090121
